src/checks/herokuLogDrain.check.js

```diff
@@ -8,6 +8,10 @@ const defaultTechnicalSummary = 'Uses the Heroku API to fetch Heroku log drains
 const defaultBusinessImpact = 'Logs may not be captured in Splunk for this application. It may not be possible to debug other issues while log drains are not configured.';
 const defaultSeverity = 2;
 
+// Required format is the Heroku app URL exlcuding the protocol and path
+// e.g. ft-next-ads-api-eu.herokuapp.com, not https:/ft-next-ads-api-eu.herokuapp.com
+const HEROKU_LOG_DRAIN_HOST_FORMAT_REGEX = new RegExp(/^(?!http:\/)[a-z\-]*\.herokuapp\.com$/);
```

Comment:
**suggestion:** because we're using a RegExp literal here with no second parameter passed into `new RegExp`, we can simplify to this:

```js
const HEROKU_LOG_DRAIN_HOST_FORMAT_REGEX = /^(?!http:\/)[a-z\-]*\.herokuapp\.com$/;
```

Also a few potential RegExp improvements:

1. We're negating the `http` part _and_ specifying a start character `^`, which makes negating the `http` redundant. We can just drop the part in the parenthesis entirely and have equivalent matches
2. I know we don't really _use_ these, but a valid hostname can include numbers too, so might be worth catering for this just in case?
3. I think `+` is more appropriate than `*` for the subdomain character set, because otherwise we'll accept `.herokuapp.com` as a valid hostname
4. When the `-` is the last character in a character set, there's no need to escape it. This is probs the most tenuous suggestion but thought it was worth noting

All together:

```suggestion
const HEROKU_LOG_DRAIN_HOST_FORMAT_REGEX = /^[a-z0-9-]+\.herokuapp\.com$/;
```

![Screenshot 2022-08-30 at 13 19 41](https://user-images.githubusercontent.com/138944/187434944-60d8db9a-a131-40ea-9c8a-ac4b40762f79.png)

Reply:
Oh actually we _do_ use numbers, I just spotted `ft-next-router-v003-us.herokuapp.com` because of recent alerts 😄

Reply:
Ooh wow - good spot! And good spots in general.

All the suggestions make total sense so will commit now.